pkg/pkgmgmt/install_test.go

```diff
@@ -75,7 +77,8 @@ func TestInstallOptions_ValidateFeedURL(t *testing.T) {
 			FeedURL: "$://example.com",
 		}
 		err := opts.validateFeedURL()
-		require.EqualError(t, err, "invalid --feed-url $://example.com: parse $://example.com: first path segment in URL cannot contain colon")
+		assert.True(t, strings.Contains(err.Error(), fmt.Sprintf("invalid --feed-url %s", opts.FeedURL)))
```

Comment:
I wanted to let you know about another testify function `assert.Contains` that can simplify your checks and improve the error message when the check fails:

```suggestion
		assert.Contains(t, err.Error(), fmt.Sprintf("invalid --feed-url %s", opts.FeedURL))
```